### PR TITLE
fix blocking of move_base_rel and add mimic support

### PR DESCRIPTION
**Pull request by floweisshardt**

### CODE REVIEW

**Comment by NadiaHG** [comment]
Visual Review: OK

File: cob_script_server/src/simple_script_server/simple_script_server.py [hunk 1]
```diff
@@ -647,6 +648,13 @@ def move_base_rel(self, component_name, parameter_name=[0,0,0], blocking=True):
 			return ah
 		else:
 			ah.set_active(mode="topic")
+			
+		parameter_topic_name = self.ns_global_prefix + "/" + component_name + "/topic_name"
+		if not rospy.has_param(parameter_topic_name):
+				rospy.logerr("parameter %s does not exist on ROS Parameter Server, aborting...",parameter_topic_name)
```
Comment (floweisshardt) [inline cob_script_server/src/simple_script_server/simple_script_server.py]: tabs vs spaces?

File: cob_script_server/src/simple_script_server/simple_script_server.py [hunk 2]
```diff
@@ -781,6 +797,78 @@ def set_light(self,component_name,parameter_name,blocking=False):
...
+		if(self.parse):
+			return ah
+		else:
+			ah.set_active(mode="topic")
+
+		rospy.loginfo("Set <<%s>> to <<%s>>", component_name, parameter_name)
+
+		service_ns = self.ns_global_prefix + "/" + component_name + "/service_ns"
+		if not rospy.has_param(service_ns):
+				rospy.logerr("parameter %s does not exist on ROS Parameter Server, aborting...",service_ns)
```
Comment (floweisshardt) [inline cob_script_server/src/simple_script_server/simple_script_server.py]: tabs vs spaces?

**Comment by floweisshardt** [comment]
visual review: OK

**Pull request merged**